ppl/zqd/db/flags.go

```diff
@@ -1,13 +1,10 @@
-package apiserver
+package db
```

Comment:
Nit: I think the methods in here should live next to their associated type declarations in `db.go`.

Reply:
I hear you, this is going to change further with helm-postgres, so I'm going to keep this as-is right now.